R/zzz.R

```diff
@@ -55,20 +55,28 @@ check_time <- function(time) {
 }
 
 
-get_widget <- function(comparison_item, category, gprop) {
+get_widget <- function(comparison_item, category, gprop, 
```

Comment:
tz and hl should be added to the main gtrends function and the passed to get_widget

Reply:
The hl was already there just it was not passed to the get_widget () internal function.  I think you should not add the tz part until it is clear how to use it. I just made it explicit, and used the preset value that was in the earlier version of get_widget().